sr_world_generator/CMakeLists.txt

```diff
@@ -0,0 +1,195 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(sr_world_generator)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED)
+
+## System dependencies are found with CMake's conventions
+# find_package(Boost REQUIRED COMPONENTS system)
+
+
+## Uncomment this if the package has a setup.py. This macro ensures
+## modules and global scripts declared therein get installed
+## See http://ros.org/doc/api/catkin/html/user_guide/setup_dot_py.html
+catkin_python_setup()
+
+################################################
+## Declare ROS messages, services and actions ##
+################################################
+
+## To declare and build messages, services or actions from within this
```

Comment:
Lots of commented lines in this file, again I don't know our policy on this. Could/should they be removed?

Reply:
I think in case of cmakelists we can leave there since there is plenty of commented out stuff that could be used in the future